src/dragonball/Cargo.toml

```diff
@@ -35,6 +35,7 @@ serde_derive = "1.0.27"
 serde_json = "1.0.9"
 slog = "2.5.2"
 slog-scope = "4.4.0"
+storage = { git = "https://github.com/dragonflyoss/image-service.git", rev = "316380792092f73c99f832c4cb44ef4319d6f76b", features = ["backend-registry","backend-oss"]}
```

Comment:
Why not use `nydus-rs = "2.1.0-alpha2"` as image-service dependencies.

Reply:
Add the comment for that. As feature backend-registry and backend-oss is not enabled in dbs crate, just enable it here temporarily

Reply:
Suggest to rise an issue.